doc/README.md

```diff
@@ -24,4 +42,4 @@ Details of reported data can be found [here](./cmd/flags/error-reporting.md)
 
 ## Issues
 
-- [Known Issues & Limitations](LIMITATIONS.md)
\ No newline at end of file
+- [Known Issues & Limitations](LIMITATIONS.md)
```

Comment:
I'm blind I don't see the diff here ...

Reply:
Only a newline missing at the end of the file added automatically sorry for the noise ...